app/forms/fields.py

```diff
@@ -1,30 +1,31 @@
 from wtforms import FormField, SelectField, SelectMultipleField, StringField
 from wtforms import validators
 
-from app.forms.custom_fields import MaxTextAreaField, CustomIntegerField
+from app.forms.custom_fields import MaxTextAreaField, CustomIntegerField, CustomDecimalField
 from app.forms.date_form import get_date_form, get_month_year_form
-from app.validation.validators import IntegerCheck, NumberRange, ResponseRequired
+from app.validation.validators import NumberCheck, NumberRange, ResponseRequired, DecimalPlaces
 from structlog import get_logger
 
 MAX_LENGTH = 2000
+MAX_NUMBER_LENGTH = 10
+MAX_DECIMAL_PLACES = MAX_NUMBER_LENGTH - 2
 logger = get_logger()
 
 
-def get_field(answer, label, error_messages):
+def get_field(answer, label, error_messages, answer_store):
```

Comment:
You're right that there's not a vulnerability here as in the context it's being used only numeric values are allowed (the `validate_range_type` test in `test_schema_validation` ensures this). 

I agree we should have a discussion about use of tainted values from the answer store - I've added a note to Tuesdays tech discussion agenda.